ibm_catalog.json

```diff
@@ -420,10 +259,48 @@
                   "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-kms-all-inclusive/main/reference-architecture/key_protect.svg",
                   "type": "image/svg+xml"
                 },
-                "description": "This architecture supports creating and configuring IBM Key Management resources"
+                "description": ""
               }
             ]
+          },
+          "dependencies": [
+          {
+            "name": "deploy-arch-ibm-account-infra-base",
+            "description": "This module prepares your IBM Cloud account with the necessary configurations to ensure a secure and organized environment for your architecture.",
+            "catalog_id": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3",
+            "flavors": [
+              "resource-group-only",
+              "resource-groups-with-account-settings"
+            ],
+            "default_flavor": "resource-group-only",
+            "id": "63641cec-6093-4b4f-b7b0-98d2f4185cd6-global",
+            "input_mapping": [
+              {
+                "dependency_output": "security_resource_group_name",
+                "version_input": "existing_resource_group_name"
+              },
+              {
+                "dependency_input": "prefix",
+                "version_input": "prefix",
+                "reference_version": true
+              },
+              {
+                "dependency_input": "provider_visibility",
+                "version_input": "provider_visibility",
+                "reference_version": true
+              },
+              {
+                "version_input": "use_existing_resource_group",
+                "value": true
+              }
+            ],
+            "optional": true,
+            "on_by_default": false,
+            "version": "v3.0.2"
           }
+        ],
+          "dependency_version_2": true,
+          "terraform_version": "1.9.0"
```

Comment:
Can you change this to "1.10.5" ? Did we check if "1.10" works?

Reply:
I haven't tested on 1.10 yet. I will update this to 1.10.5 and do the testing. FYI @daniel-butler-irl